test/auth.test.js

```diff
@@ -115,6 +116,24 @@ describe('Auth linking', () => {
     expect(newIdentities[1].provider_type).toEqual('local-userpass');
   });
 
+  it('should link two accounts without sending a confirmation email', async() => {
```

Comment:
this test string seems very confusing, it seems like this test is testing that you can link a user that was confirmed via `confirmByEmail` the string should more accurately represent that